helm/conjur-config-cluster-prep/sample-values.yaml

```diff
@@ -0,0 +1,11 @@
+# User-supplied values for the conjur-config-cluster-prep Helm chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+conjur:
+  account: "<Insert Conjur Account Here>"
+  applianceUrl: "https://insert.conjur.appliance.url.here"
```

Comment:
If we're going to be doing any post-processing of the generated, raw manifests, it would be helpful to wrap the `applianceUrl` and `certificateBase64` canned values in angled brackets.

Otherwise, I'm good with the canned values used here if @alexkalish is.

Reply:
Angled brackets for all canned values would be ideal, but the schema we set for the url field for example will complain if the value isn't valid. Maybe there's a way to skip the schema validation?

Reply:
I don't know of an easy way to skip the schema validation... however, this sequence might work:

- Run helm template to generate raw manifests
- Run kustomize to fix up the "fill-in-the-blank" values in the generated raw manifests.

Reply:
Used `yq` instead of `kustomize` since the latter does not preserve comments